src/components/custom/dashboard/client/pages/diet-tracker/DayNavigation.jsx

```
@@ -0,0 +1,339 @@
+              <div
+                key={dayNumber}
+                className={getDayCardStyle(dayNumber)}
+                onClick={() => onSelectDay(dayNumber)}
+              >
+                {/* Progress Ring - Fixed positioning */}
+                <div className="absolute top-3 right-3 z-10">
+                  <div className="relative w-9 h-9">
+                    <svg
+                      className="w-9 h-9 transform -rotate-90"
+                      viewBox="0 0 36 36"
+                    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)